models/util.go

```diff
@@ -94,3 +94,9 @@ func CurlyBraces(value string) string {
 		return value
 	}
 }
+
+func GetMORnPrefix(dn string) string {
```

Comment:
Does this function work for Rns containing '/'?

Reply:
![Screenshot 2023-09-12 at 9 40 44 AM](https://github.com/ciscoecosystem/aci-go-client/assets/23535091/bba0a3f4-8e3e-4b2a-9e3e-76c9c23c97e0)